make_lua_ffi.pl

```diff
@@ -66,6 +70,9 @@ sub replace_keywords {
 			$line =~ s/\b$keyword\b/${keyword}_/g;
 		}
 	}
+	foreach my $keyword (@c11_keywords_ignored) {
+		$line =~ s/\b$keyword\b//g;
```

Comment:
May be it makes sense to add comment in `$output` like it's done in other cases within this function?

Reply:
/* keyword */ ?
I don't comment out the whole line, just one keyword.

Reply:
Ah, okay, then let's comment this keyword out in the output instead of stripping it completely.